sdk/core/azure_core_amqp/README.md

```diff
@@ -6,6 +6,39 @@ Azure AMQP crate for consumption of AMQP based packages in the Azure SDK for Rus
 
 This crate is part of a collection of crates: for more information please refer to [https://github.com/azure/azure-sdk-for-rust](https://github.com/azure/azure-sdk-for-rust).
 
+## SOCKS5 Proxy Support
+
+This crate supports SOCKS5 proxy connections for corporate environments. SOCKS5 support is enabled by configuring the `custom_endpoint` option with a SOCKS5 URL:
+
+```rust,no_run
+use azure_core::http::Url;
+use azure_core_amqp::AmqpConnectionOptions;
+
+# fn main() -> Result<(), Box<dyn std::error::Error>> {
+let mut options = AmqpConnectionOptions::default();
+options.custom_endpoint = Some(Url::parse("socks5h://proxy.corp.com:8080")?);
```

Comment:
We generally prefer the construction pattern:

```rust
let options = AmqpConnectionOptions {
  custom_endpoint = Some("socks5h://proxy.contoso.com:8080".parse()?),
  ..Default::default()
};
```

And use contoso.com or example.com.

Reply:
Also means `azure_core::http::Url` doesn't need to be imported.